packages/expo/src/launch/RootErrorBoundary.tsx

```diff
@@ -5,7 +5,7 @@ import { getAppLoadingLifecycleEmitter } from './AppLoading';
 
 const { ExponentAppLoadingManager } = NativeModules;
 
-async function finishedAsync() {
+async function finishedAsync(): Promise<any> {
```

Comment:
Can remove this function entirely, actually, since the oldest caller is in SDK 28.